<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartStopRangeWidget</class>
 <widget class="QWidget" name="StartStopRangeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>191</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>15</number>
   </property>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="stopValueLayout"/>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="startValueLayout"/>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="centerValueLayout"/>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="spanValueLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
